rest-service/manager_rest/rest/resources_v1/events.py

```diff
@@ -51,38 +54,98 @@ class Events(SecuredResource):
     DEFAULT_SEARCH_SIZE = 10000
 
     @staticmethod
-    def _build_select_query(_include, filters, pagination, sort):
+    def _apply_range_filters(query, model, range_filters):
+        """Apply range filters to query.
+
+        :param query: Query in which the filtering should be applied
+        :type query: :class:`sqlalchemy.orm.query.Query`
+        :param model: Model to use to apply the filtering
+        :type model:
+            :class:`manager_rest.storage.resource_models.Event`
+            :class:`manager_rest.storage.resource_models.Log`
+        :param range_filters: Range filters passed as a request argument
+        :type range_filters: dict(str, dict(str))
+        :returns: Query with filtering applied
+        :rtype: :class:`sqlalchemy.orm.query.Query`
+
+        """
+        for field, range_filter in range_filters:
+            # Drop `@` prefix for compatibility
+            # with old Elasticsearch based implementation
+            field = field.lstrip('@')
+            if hasattr(model, field):
+                query = Events._apply_range_filters(
+                    query, model, field, range_filter)
+        return query
+
+    @staticmethod
+    def _apply_range_filter(query, model, field, range_filter):
+        """Apply a range filter to query.
+
+        :param query: Query in which the filtering should be applied
+        :type query: :class:`sqlalchemy.orm.query.Query`
+        :param model: Model to use to apply the filtering
+        :type model:
+            :class:`manager_rest.storage.resource_models.Event`
+            :class:`manager_rest.storage.resource_models.Log`
+        :param field: Field in the model that should be filtered
+        :type field: str
+        :param range_filter: Range filter passed as a request argument
+        :type range_filter: dict(str)
+        :returns: Query with filtering applied
+        :rtype: :class:`sqlalchemy.orm.query.Query`
+
+        """
+        if 'from' in range_filter:
```

Comment:
From this method's point of view:
- when `from` is missing, all records that are oldest than the timestamp in `to` will be retrieved
- when `to` is missing, all records that are newer than the timestamp in `from` will be retrieved
- when both `from` and `to` are missin, all records will be retrieved (no filtering applied)

However, from the [`rangeable`](https://github.com/cloudify-cosmo/cloudify-manager/blob/CFY-6332-fix-events-integration-tests/rest-service/manager_rest/rest/rest_decorators.py#L206) decorator point of view, all three values (field name, from and to) must be present.

What might be missing here is that rangeable doesn't enforce any validation of the values in `from` and `to`, which makes sense because the might be numbers, dates or some other thing. However, the decorator can be probably expanded to pass a validation function to make sure that `from` and `to` have the right format. In any case, this is out of the scope of the current PR, so I'll do that in a separate one.